test/functional/feature_signet.py

```diff
@@ -50,14 +67,24 @@ def setup_network(self):
     def run_test(self):
         self.log.info("basic tests using OP_TRUE challenge")
 
+        self.log.info('getblockchaininfo')
+        check_getblockchaininfo(self.nodes[1], self.signets[0])
+        check_getblockchaininfo(self.nodes[2], self.signets[1])
+        check_getblockchaininfo(self.nodes[5], self.signets[2])
+
         self.log.info('getmininginfo')
-        mining_info = self.nodes[0].getmininginfo()
-        assert_equal(mining_info['blocks'], 0)
-        assert_equal(mining_info['chain'], 'signet')
-        assert 'currentblocktx' not in mining_info
-        assert 'currentblockweight' not in mining_info
-        assert_equal(mining_info['networkhashps'], Decimal('0'))
-        assert_equal(mining_info['pooledtx'], 0)
+        def check_getmininginfo(node_idx, signet_idx):
```

Comment:
Thanks for the adding the helper. But now it's inconsistent with the `check_getblockchaininfo` helper. No strong preference for which style to use.

Maybe make it `def check_getmininginfo(self, node_idx, signet_idx):` and `check_getblockchaininfo(self, node_idx, signet_idx) ` and put them above `run_test`.

Reply:
Both helper functions are internal to `run_test` in ecaa786cc103cf7cc63ae899ec13d81a54e2fd1e.
`check_getmininginfo` makes specific assertions on values for fields such as `blocks`, which may change during a test. It is best to keep it as localized as possible, since it is unlikely to be broadly useful.